client/js/backstage.js

```diff
@@ -157,29 +157,29 @@ var BackstageView = Backbone.View.extend({
         this.$openButton    = this.$el.find('.silky-bs-open-button')
         this.$saveButton    = this.$el.find('.silky-bs-save-button')
         this.$saveAsButton  = this.$el.find('.silky-bs-save-as-button')
-		
+
 		this.main = new BackstagePlaces({ el: ".silky-bs-main", model: this.model })
 	},
 	activate : function() {
-	
+
 		this.$el.fadeIn(100)
-		
```

Comment:
your text-editor has strong ideas that empty lines shouldn't have whitespace, which is probably OK, but it does mean you're committing a lot of files that haven't really changed, and makes code review a bit of a pain for me. but i can live with it, if there's no easy way to turn it off.